src/backend/updating-llvm.md

```diff
@@ -119,12 +119,12 @@ through each in detail.
    Ideally, build LLVM and test it on a few platforms:
 
    * Linux
-   * OSX
+   * macOS
    * Windows
 
    and afterwards run some docker containers that CI also does:
 
-   * `./src/ci/docker/run.sh wasm32-unknown`
+   * `./src/ci/docker/run.sh wasm32`
```

Comment:
Why did this change?

Reply:
The dir name is changed, it should be matched here: https://github.com/rust-lang/rust/tree/master/src/ci/docker/host-x86_64